packages/rule-schema-to-typescript-types/src/index.ts

```diff
@@ -50,7 +50,7 @@ export async function compile(
     return await prettier.format(unformattedCode, await prettierConfig);
   } catch (e) {
     if (e instanceof Error) {
-      e.message = e.message + `\n\nUnformatted Code:\n${unformattedCode}`;
+      e.message = `${e.message}\n\nUnformatted Code:\n${unformattedCode}`;
```

Comment:
Collides with `operator-assignment`?

Reply:
yeah, I'll resolve the merge conflicts once one is merged